mentors/csharp/s.json

```diff
@@ -22,5 +22,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "C# developer with a focus on QA Test Automation as well as Automation in general."
+  },
+  {
+    "github_username": "ShahryarSaljoughi",
+    "name": "Shahryar Saljoughi",
+    "line_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I'm a CS student, who uses likec c# for its power."
```

Comment:
Hello Shahryar!

I think words twisted in this sentence, can you clear them up a bit?

Reply:
Hi, 
Sorry for that. I've corrected that sentence and made a new pull request.